server/container/guice/mailbox/src/main/java/org/apache/james/modules/MailboxProbeImpl.java

```diff
@@ -93,6 +93,18 @@ public Mailbox getMailbox(String namespace, String user, String name) {
         }
     }
 
+    public boolean exist(MailboxPath path) {
```

Comment:
for me this commit (6dbb2f6a53ad761bfc1b79835f0936f9569c935d  JAMES-2469 Parallelize JMAP integration test provisioning where possible ) is going too far. Does parallelizing provisionning realy realy improve so much test duration?

Reply:
~50 ls on  tests that takes down to 700ms represents a ~14% testing time reduction, which is a good stat in the end.

Furthermore, given the `runParallel` tool I find the complexity of this commit reasonable.